src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
according to the naming convention enum should be singular
```suggestion
public enum Colors{
```